base-local/database/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mariadb:latest
+
+RUN  apt-get update \
+  && apt-get install -y wget \
+  && rm -rf /var/lib/apt/lists/*
+
+COPY ./scripts/01_create_db_and_users.sql /docker-entrypoint-initdb.d/
+
+RUN wget https://raw.githubusercontent.com/readium/readium-lcp-server/master/dbmodel/mysql_db_setup_lcpserver.sql -O /docker-entrypoint-initdb.d/02_mysql_db_setup_lcpserver.sql \
+  && sed -i '1 s/^/USE lcpserver;\n/' /docker-entrypoint-initdb.d/02_mysql_db_setup_lcpserver.sql \
+  && wget https://raw.githubusercontent.com/readium/readium-lcp-server/master/dbmodel/mysql_db_setup_lsdserver.sql -O /docker-entrypoint-initdb.d/03_mysql_db_setup_lsdserver.sql \
+  && sed -i '1 s/^/USE lsdserver;\n/' /docker-entrypoint-initdb.d/03_mysql_db_setup_lsdserver.sql \
+  && wget https://raw.githubusercontent.com/readium/readium-lcp-server/master/dbmodel/mysql_db_setup_frontend.sql -O /docker-entrypoint-initdb.d/04_mysql_db_setup_frontend.sql \
+  && sed -i '1 s/^/USE testfrontend;\n/' /docker-entrypoint-initdb.d/04_mysql_db_setup_frontend.sql
```

Comment:
This is all good. It works much better. Question: should be add a shell script that does the "secure installation" bits?

Reply:
Do you want to move those 6 lines to a bash script and run it inside Dockerfile?

Reply:
Or would you like to use `MYSQL_ONETIME_PASSWORD` or `MYSQL_RANDOM_ROOT_PASSWORD`? In this case we we would need to use different credentials for `lcpserver`, `lsdserver` and `testfrontend` databases

Reply:
For now let's leave it. It's something I want to keep in mind for a future iteration somewhere, where a deployment might possibly be used in production.